docs/additional-links/nimbus-devtools-guide.md

```diff
@@ -0,0 +1,106 @@
+---
+id: nimbus-devtools-guide
+title: Nimbus Developer Tools Guide
+slug: /nimbus-devtools-guide
+---
+
+# Nimbus Developer Tools Guide
+
+This guide provides an overview of the Nimbus Developer Tools, designed for Nimbus experiment debugging. With these tools, you can more easily test experiments, debug targeting expressions, and browse through available experiments.
+
+### Installation
+
+To install the Nimbus Developer Tools, download the latest release from the [Nimbus DevTools GitHub repository](https://github.com/mozilla-extensions/nimbus-devtools/releases/tag/release%2Fv0.1.0).
```

Comment:
This should link to `https://github.com/mozilla-extensions/nimbus-devtools/releases` otherwise we need to update this link with every new version.